docs/service-release-checklist.md

```diff
@@ -0,0 +1,104 @@
+# Service readiness checklist
+
+This document aims to provide a checklist for determining whether service is mature enough for general availability.
+
+## Checklist
+
+Legend: `M` - Mandatory, `R` - Recommended, `O` - Optional
+
+### Documentation
+
+- `R`: README.md in the repository:
+  * service description, ownership, links to documentation,
+    build and deployment instructions, configuration variables, healthchecks, etc
+- `R`: Architectural diagram for the service
+- `O`: SLI/SLO/SLA definitions (at least service criticality level: P1/2/3)
+
+### Build and release pipeline
+
+- `M`: GitHub Actions worklows for building the code, running linting check and executing tests
+- `M`: Artifacts are container images
```

Comment:
We should explicitly state that they are docker images, we don't support other containerization platform.

Reply:
this is public version of template, people can use whatever they like, these are broad recommendations. we should clarify our internal ones, though